### ISSUE TITLE
Issue running cask installers which prompt for a password

### ISSUE THREAD

**Issue by matthew-cox** [Homebrew/homebrew-cask#98092]
Hello!

I'm running into an issue which seems to be limited to casks that use an installer which expects to be able to prompt for a password. The `brew bundle` task hangs until a SIGINT (Control-C) and only after that does it display the password prompt.

For example:

```
Installing ftdi-vcp-driver cask. It is not currently installed.
==> Caveats
If you don't want to reboot, you can load the driver using the following
command:

  sudo /sbin/kextload -b com.FTDI.driver.FTDIUSBSerialDriver

Once you've rebooted or loaded the driver, you can connect your FTDI
based cable to a USB port and it will show up in /dev, usually like this:

  /dev/tty.usbserial-XXXXXXXX

where XXXXXXXX is a random ID, based on the serial number of your FTDI
based cable.

NOTE: If your FTDI based cable was already connected before you installed
the driver, you'll need to unplug the cable from the USB port and
reconnect it for it to show up in /dev.

ftdi-vcp-driver requires a kernel extension to work.
If the installation fails, retry after you enable it in:
  System Preferences → Security & Privacy → General

For more information, refer to vendor documentation or this Apple Technical Note:
  https://developer.apple.com/library/content/technotes/tn2459/_index.html

You must reboot for the installation of ftdi-vcp-driver to take effect.

==> Downloading https://www.ftdichip.com/Drivers/VCP/MacOSX/FTDIUSBSerialDriver_v2_4_4.dmg
######################################################################## 100.0%
==> Installing Cask ftdi-vcp-driver
==> Running installer for ftdi-vcp-driver; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are ignored.


^C

Password:
Error: SIGHUP
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/bundle.rb:8:in `initialize'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/bundle.rb:8:in `initialize'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/bundle.rb:8:in `system'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/bundle.rb:8:in `system'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/cask_installer.rb:38:in `install'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/installer.rb:35:in `block in install'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/installer.rb:11:in `each'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/installer.rb:11:in `install'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/commands/install.rb:10:in `run'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/cmd/bundle.rb:94:in `bundle'
/usr/local/Homebrew/Library/Homebrew/brew.r%                                                                                                                                 b:124:in `<main>'
```

If I re-run with the same Brewfile, it seems that the cask is installed:

```
❯ brew bundle --file ~/.config/zero/dotfiles/workspaces/shared/Brewfile
...
Using fluid
Using ftdi-vcp-driver

❯ brew info --cask ftdi-vcp-driver
ftdi-vcp-driver: 2.4.4
https://www.ftdichip.com/Drivers/VCP.htm
/usr/local/Caskroom/ftdi-vcp-driver/2.4.4 (60.4KB)
From: https://github.com/Homebrew/homebrew-cask-drivers/blob/HEAD/Casks/ftdi-vcp-driver.rb
==> Name
FTDI VCP Driver
==> Description
None
==> Artifacts
FTDIUSBSerial.pkg (Pkg)
==> Caveats
```

However, I'm not certain if this cask would be setup correctly as I never actually entered my password.

I'll note that other casks prompt for a password (maybe just a sudo prompt) correctly:

```
...
Installing hex-fiend
Password:
...
```
Let me know how I can help further!

Thank you,
Matt

**Comment by vitorgalvao** [comment]
A `hex-fiend` installation shouldn’t be asking for your password. And are you able to give other casks what this happens?

Finally, please edit the issue to include *everything* from [the issue template](https://github.com/Homebrew/homebrew-cask/blob/master/.github/ISSUE_TEMPLATE/01_bug_report.md). If you’d rather close this and open a new issue, that’s fine.

**Comment by vitorgalvao** [comment]
Closing for lack of reply.
